hw2/05-form.js

```diff
@@ -1 +1,51 @@
-// Add your code here
+const form = document.querySelector("form"); // Get form
+form.addEventListener("submit", handleSubmit); // Get input on submit
+
+// Extract form information and output in a modal
+function handleSubmit(event) {
+  event.preventDefault();
+
+  // Extract info from form
+  const dataName = document.getElementById("name").value.trim();
+  console.log(`Data Name: ${dataName}`);
+  const dataEmail = document.getElementById("email").value.trim();
+  console.log(`Data Email: ${dataEmail}`);
+  const dataRegistration = document.getElementById("registration").value;
+  console.log(`Registered: ${dataRegistration}`);
+  const pLang = document.getElementById("pLang");
+  console.log(`Programming Languages: ${pLang}`);
+  const OS = document.getElementById("os");
+  console.log(`Operating Systems: ${OS}`);
+  const fullStack = document.getElementById("full-stack");
+  console.log(`Full Stack: ${fullStack}`);
+  const additional = document.getElementById("questions");
+  console.log(`Other Comments: ${additional}`);
+}
+
+// Extract form information
+// const name = document.createElement("p");
+// const name = form.elements.name.type;
+// console.log(name);
+
+// Create Modal
```

Comment:
This version of [Boostrap](https://getbootstrap.com/docs/5.2/components/modal/) supports using vanilla JS to directly handle and manipulate the Modal. You can use methods like .show and .hide in this situation instead of creating your own. If you do go with this approach, then you would also need the script for JS to be able to use it.

Reply:
Thanks for the heads up!  I have been stuck on this part and spent so many hours just connecting it up lol.  I will try that.